workshops/CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+2. Then join the `#curriculum` channel: [https://starthackclub.slack.com/messages/curriculum/](https://starthackclub.slack.com/messages/curriculum/)
+3. Ask your question!
+
+## How to Contribute to the Workshops
+
+There are three ways to contribute:
+
+1. [Create an Issue](#file-an-issue)
+2. [Improve an existing workshop](#improve-an-existing-workshop)
+3. [Create a new workshop](#create-a-new-workshop)
+
+### File an Issue
+
+If you notice any problems or have an idea you want to exist, you can create a
+GitHub issue describing the improvement you want to see. Click
+[here](https://github.com/hackclub/hackclub/issues/new) to create a new GitHub
+issue.
+
+### Improve an Existing Workshop
+
+If you want to go further, you can then submit a pull request with the changes
+you want to see. A walkthrough of how to submit a pull request can be found
+[here](../CONTRIBUTING.md).
+
+### Create a New Workshop
+
+We also love contributions of new workshops! Make sure you've read through
+[`CONTRIBUTING.md`](../CONTRIBUTING.md)
+
+Here are our requirements for making a workshop:
+
+**Step 1: Creating the necessary files and folders:**
+
+Before continuing, browse through the directory structure of some
+workshops like [Portfolio](portfolio) and [Dodge](dodge) before continuing.
+
+1. Make sure that you've followed the directions in
+   [`CONTRIBUTING.md`](../CONTRIBUTING.md) and cloned the `hackclub` repo.
+2. Create a directory with the name of the workshop in the
+   [`/workshops`](../workshops) directory. The name of your workshop should only
+   be one or two words like `soccer` or `remote_control`.
+3. Inside the directory you just created, create a `README.md` file.
+   You will be writing your workshop in this Markdown file. Feel free to create
+   other Markdown files that you link to in your `README.md`. The workshop
+   [Dodge](dodge) for example does this.
+4. From [`/workshops/README.md`](README.md) add your workshop to the
+   [Community Workshops](README.md#community-workshops) table.
+
+**Step 2: Create the workshop with the following requirements:**
+
+- A learner can finish a workshop in 1 hour or less.
+- By the end of the workshop, a learner will have finished building something
+  that they can show their friends.
+- Workshops are written in Markdown
+- Workshops use GIFs to explain how to do things
+- Workshops are written assuming that the learner will be following along in
+  [Cloud9](https://c9.io), an online IDE. We do this
+  so that anyone with a standard netbook and an internet connection can use
+  our curriculum
+- The workshop will have checkpoints in it that show working code examples
+  inside of a [JS Bin](https://jsbin.com) examples. To create these JS Bins,
+  use a tool we created, `jsbinctl`. Here is a link to
+  [`jsbinctl`'s README](lib/jsbinctl/README.md)
+
+
+
+**Intellectual property ownership of the workshop:**
+
+Under the [liscence](TODO) we have, your work would be open sourced
```

Comment:
By the way, you don't need to comment when you fix things because the diff will be outdated and GitHub will mark it as such :-).